lib/vagrant-service-manager/command.rb

```diff
@@ -192,14 +175,14 @@ def execute_docker_info
           # First, get the TLS Certificates, if needed
           if !File.directory?(secrets_path) then
 
-            # Get the private key
-            hprivate_key_path = machine.ssh_info[:private_key_path][0]
+            # Create the directory
+            Dir.mkdir(secrets_path)
```

Comment:
Do we need to explicitly create a new directory? Did you check if `machine.communicate.download` creates one if not already exist ?